examples/beautiful_mnist.py

```diff
@@ -21,7 +21,7 @@ def __call__(self, x:Tensor) -> Tensor: return x.sequential(self.layers)
   X_train, Y_train, X_test, Y_test = mnist(fashion=getenv("FASHION"))
 
   model = Model()
-  opt = nn.optim.Adam(nn.state.get_parameters(model))
+  opt = nn.optim.Adam(nn.state.get_parameters(model)) if not getenv("MUON") else nn.optim.Muon(nn.state.get_parameters(model))
```

Comment:
opt = (nn.optim.Adam if not getenv("MUON") else nn.optim.Muon)(nn.state.get_parameters(model))